app/assets/stylesheets/parent_objects.scss

```diff
@@ -99,7 +112,6 @@ DEFAULT DESKTOP STYLING
 
 #parent-show table {
```

Comment:
You can delete this because you removed the 'grid template' in the containing div so the 'grid area' isn't doing anything anymore.